packages/iov-keycontrol/src/userprofile.ts

```diff
@@ -191,9 +191,9 @@ export class UserProfile {
   }
 
   // creates an identitiy in the n-th keyring entry of the primary keyring
-  public async createIdentity(id: number | KeyringEntryId): Promise<LocalIdentity> {
+  public async createIdentity(id: number | KeyringEntryId, options?: any): Promise<LocalIdentity> {
```

Comment:
not so happy with the `any|undefined` type, as mentioned in another issue (let me know if you want me to find the link), but let's clean that up in 0.7. For now we need for compatibility.

Reply:
I am also not exactly happy. But I prefer to use any until we know what kind of options we'll have in 0.7. First step is to make options non-optional and then we see what is left